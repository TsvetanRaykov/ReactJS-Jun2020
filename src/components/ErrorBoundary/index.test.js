import React from 'react'
import renderer from 'react-test-renderer'
import TestWrapper from '../../utils/test-wrapper'
import ErrorBoundary from '.'

jest.mock('react', () => ({
	...jest.requireActual('react'),
	useLayoutEffect: jest.requireActual('react').useEffect,
}))

describe('<ErrorBoundary />', () => {
	it('should match the snapshot', () => {
		const wrapper = renderer.create(
			<TestWrapper>
				<ErrorBoundary>
					<h1>TEST</h1>
				</ErrorBoundary>
				)
			</TestWrapper>
		)
		expect(wrapper).toMatchSnapshot()
	})
})
