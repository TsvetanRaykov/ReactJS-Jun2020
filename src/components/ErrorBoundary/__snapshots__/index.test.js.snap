// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBoundary /> should match the snapshot 1`] = `
Array [
  <h1>
    TEST
  </h1>,
  ")",
]
`;
